components/camera/videosource/webcam.go

```diff
@@ -136,13 +136,11 @@ func getProperties(d driver.Driver) (_ []prop.Media, err error) {
 type WebcamAttrs struct {
 	CameraParameters     *transform.PinholeCameraIntrinsics `json:"intrinsic_parameters,omitempty"`
 	DistortionParameters *transform.BrownConrady            `json:"distortion_parameters,omitempty"`
-	Stream               string                             `json:"stream"`
 	Debug                bool                               `json:"debug,omitempty"`
-	Format               string                             `json:"format"`
-	Path                 string                             `json:"video_path"`
-	PathPattern          string                             `json:"video_path_pattern"`
-	Width                int                                `json:"width_px"`
-	Height               int                                `json:"height_px"`
+	Format               string                             `json:"format,omitempty"`
+	Path                 string                             `json:"video_path,omitempty"`
```

Comment:
I think with "Shown but not required" this means that this should NOT be omitempty. 

@Fahmina or @nataliajacobowitz do you know how it will end up looking?